pkg/container-runtime/driver.go

```diff
@@ -0,0 +1,76 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package container_runtime
+
+import (
+	"github.com/sealerio/sealer/pkg/infradriver"
+	"net"
+)
+
+type ConfigForRegistry struct {
```

Comment:
Why should ContainerRuntimeInstaller know ConfigForRegistry?

Reply:
this is not for `ContainerRuntimeInstaller`, its `Configurator` for ContainerRuntime, for example: config ContainerRuntime to access local registry.